### PR TITLE
Migrate documentation to VitePress and main GitHub Pages website 

### PR DESCRIPTION
**Pull request by raucao**
See https://community.remotestorage.io/t/new-website-and-rs-js-documentation-site/850 for details and https://rs-website-preview.5apps.com/ for current live preview.

refs #1201 #1200

### CODE REVIEW

File: docs/cordova.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 within wrappers targeted to each platform, and rely on
-standards-compliant API bindings to access each device's capabilities
-such as sensors, data, network status, etc. [#f1]_
+standards-compliant API bindings to access each device\'s capabilities
+such as sensors, data, network status, etc.
 
-To use remoteStorage.js in a Cordova app, you need to have the `InAppBrowser plugin
-<https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/>`_
+To use remoteStorage.js in a Cordova app, you need to have the
+[InAppBrowser
```
Comment (DougReeder) [inline docs/cordova.md:11]: Does this line break need to be removed?

File: package.json [hunk 2]
```diff
@@ -29,7 +29,10 @@
     "preversion": "npm test && npm run build:js",
     "version": "npm run build:release && git add release/ && npm run update-doc-version && git add doc/version.py",
     "update-doc-version": "bash scripts/update-doc-version.sh",
-    "autobuild-docs": "cd doc && sphinx-autobuild . _build/html"
+    "autobuild-docs": "cd doc && sphinx-autobuild . _build/html",
```
Comment (DougReeder) [inline package.json:32]: We should remove the build:docs and autobuild:docs scripts if we're not going to use them.

**Review by DougReeder — APPROVED** [review]
LGTM

File: docs/cordova.md (hunk 1 above)
Comment (raucao) [reply docs/cordova.md:11]: No, it's in line with the width constraints of the rest of the text.

**Pull request merged**